crates/project_panel/src/project_panel.rs

```diff
@@ -943,6 +943,9 @@ impl ProjectPanel {
         self.edit_state = None;
         self.update_visible_entries(None, cx);
         self.marked_entries.clear();
+        if cx.has_active_drag() {
```

Comment:
I'm somewhat surprised that this works: project panel is the file view on the right, but we also may drag and drop into assistant panel, terminal panel and the editor itself (center pane).

Usually, the event handling is focus-based, but turns out that even the closed project panel reacts to this.

I am still not certain this is a good thing: let's add this handler to the global thing called `Workspace` (== current window for simplicity)

here are its handlers: 
https://github.com/zed-industries/zed/blob/db8cb45527b2a8fa3ed823bfccb3b8e662a640f4/crates/workspace/src/workspace.rs#L328

and we can add another one

Reply:
Hmmm, My solution is quite ugly, I will to to find a new way.